doc/api/assert.md

```diff
@@ -767,17 +770,41 @@ assert.throws(
 
 Note that `error` can not be a string. If a string is provided as the second
 argument, then `error` is assumed to be omitted and the string will be used for
-`message` instead. This can lead to easy-to-miss mistakes:
+`message` instead. This can lead to easy-to-miss mistakes. Please read the
+example below carefully if you intend to use a string as second argument:
```

Comment:
`as a second argument`

Reply:
I would think it should be more like `as the second argument` since there can only be one